raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -108,7 +113,7 @@ library ChannelManagerLibrary {
         address channel_address,
         uint caller_index,
         uint partner_index)
-        private
+        private returns (bool success)
```

Comment:
No need for return value here. There is no way this can fail in the way it is currently used.
You are using it only inside an if check for whether a channel exists or not. So we defnitely know that a channel exists. Deleting the channel once it exists, can't fail in any possible way.
 It can throw, but it can not ever return false, so there is no reason for the return value here.